### ISSUE TITLE
[Feature] Function to remove/clear Github Notifications

### ISSUE THREAD

**Issue by wolfposd** [ForkIssues/Tracker#1167]
Would be great if I could remove notifcations or clear the notifications-tab completely
Or have notifications be automatically removed once I have read them.

Currently it shows all of them.

like so:
![grafik](https://user-images.githubusercontent.com/2557290/97448004-74955380-1930-11eb-9aa6-ef44010cbde4.png)

**Comment by YesThatAllen** [comment]
Agreed. Also, I'd love an option to auto-delete a notification when I click to open it.
